lib/src/base-styles/_nx-typography.scss

```diff
@@ -7,6 +7,27 @@
 @import '../scss-shared/nx-container-helpers';
 @import '../scss-shared/nx-text-helpers';
 
+@font-face {
+  font-family: "Open Sans";
+  src: url("../assets/fonts/open-sans-v17-latin-regular.woff") format("woff");
+  font-weight: normal;
+  font-style: normal;
+}
+
+@font-face {
+  font-family: "Open Sans";
+  src: url("../assets/fonts/open-sans-v17-latin-600.woff") format("woff");
+  font-weight: 600;
+  font-style: normal;
+}
+
+@font-face {
+  font-family: "Open Sans";
+  src: url("../assets/fonts/open-sans-v17-latin-700.woff") format("woff");
+  font-weight: bold;
+  font-style: normal;
+}
```

Comment:
No italic styles?

Reply:
> No italic styles?

https://github.com/sonatype/sonatype-react-shared-components/pull/4/commits/13c13e0ed36b7679ec534556943584bcb28208ed

Reply:
> 13c13e0

No _**bold italic**_ styles? :grin:

Reply:
> > [13c13e0](https://github.com/sonatype/sonatype-react-shared-components/commit/13c13e0ed36b7679ec534556943584bcb28208ed)
> 
> No _**bold italic**_ styles? 😁

LOL

Reply:
> > [13c13e0](https://github.com/sonatype/sonatype-react-shared-components/commit/13c13e0ed36b7679ec534556943584bcb28208ed)
> 
> No _**bold italic**_ styles? 😁

Well no, we have definitely never had specific bold italic fonts installed. 
BUT if the feeling is that we should have then should we also consider adding Light, Light Italic, Semi-bold Italic? Just to be complete?

The Extra Bold is really ugly IMO, no way should we add that.